src/main/java/Bob.java

```diff
@@ -0,0 +1,338 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.Path;
+import org.apache.commons.lang3.StringUtils;
+
+public class Bob {
+    private final Scanner scanner = new Scanner(System.in);
+
```

Comment:
inconsistent indentation? Maybe can either put everything with newline or without

Reply:
Thanks! I have standardized the spacings